tests/regression/lp-1802581/mock-gpio.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/python3
+
+import os
+import selectors
+import sys
+
+
+def data_to_gpio(data):
+    return "gpio"+data.decode().strip()
+
+
+def export_ready(fd, mask):
+    data = os.read(fd, 128)
+    # allow quit
+    if data == b'quit\n':
+        return False
+    with open(data_to_gpio(data), "w"):
+        pass
+    return True
+    
+
+def unexport_ready(fd, mask):
+    os.remove(data_to_gpio(os.read(fd, 128)))
+    return True
+
+
+def dispatch(sel):
+    for key, mask in sel.select():
+        callback = key.data
+        if not callback(key.fileobj, mask):
+            return False
+    return True
+
+
+if __name__ == "__main__":
+    os.chdir(sys.argv[1])
```

Comment:
Could you please move this to `main`. Doing it like this makes `sel`, `efd` and `ufd` globals that can silently be picked up from typos elsewhere in the code.